isisd/isis_adjacency.c

```diff
@@ -328,15 +328,14 @@ void isis_adj_state_change(struct isis_adjacency **padj,
 				circuit->upadjcount[level - 1]--;
 				if (circuit->upadjcount[level - 1] == 0)
 					isis_tx_queue_clean(circuit->tx_queue);
-
-				hook_call(isis_adj_state_change_hook, adj);
```

Comment:
need to remove `hook_call()` for broadcast interfaces as well (both L1 and L2)

Reply:
Ok, will correct this!